Projects/Server/Main.cs

```diff
@@ -359,6 +360,11 @@ public static void Main(string[] args)
             AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
             AppDomain.CurrentDomain.ProcessExit += CurrentDomain_ProcessExit;
 
+            Log.Logger = new LoggerConfiguration()
+                .WriteTo.Console(
+                    outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}] {Message:lj} <s:{SourceContext}>{NewLine}{Exception}")
+                .CreateLogger();
```

Comment:
I already wrote some code that makes this configurable from a json file using a similar pattern as `ServerConfiguration.cs`, `EmailConfiguration.cs` etc.. is it ok to add it in scope for this change ?

Reply:
I would try to get this PR merged then move the configuration to json, what do you think?